### ISSUE TITLE
Hide irrelevant cleaners root vs user

### ISSUE THREAD

**Issue by soundchekk** [bleachbit/bleachbit#294]
Ubuntu 17.10
BB 1.21

The ticket refers to the function "Hide irrelevant cleaners"
I'd expect different cleaners for root and normal users.

If I run bb as normal user I'd expect e.g. 
"APT" 
"Journald" 
and certain "System" cleaners e.g. logrotate,memory,temporary files

to be irrelevant.

Above cleaners are not working if bb is being run as normal user. Thus these might be 
considered irrelevant.  bb obviously is reporting errors when selecting these. 

If you run bb as root you can flip the whole subject. Just list above as relevant only.

**Comment by az0** [comment]
Yes, I see. With APT, journald, and others it is possible to predict whether the user has permissions.

The original and current meaning of "irrelevant" in this system is whether there is anything to be cleaned, not so much about the permissions. Do you think by hiding these options that users would not understand they have an opportunity to also run BleachBit as root?

**Comment by soundchekk** [comment]
You could offer a user and a admin (root) mode with related cleaners being
selected from the GUI.
Or if admin/root cleaners are selected from the big list you could ask for
a root password before executing them instead of running into errors.
You might rearrange the cleaner grouping in admin/root and standard user
groups. Avoid the mixing.

For sure users ( that includes me) would appreciate a much easier root/user
handling and won't be confused by looking at error messages anymore.
